mt-kahypar/partition/coarsening/multilevel_uncoarsener.cpp

```diff
@@ -47,9 +48,15 @@ namespace mt_kahypar {
                                               _context.partition.verbose_output &&
                                               _context.partition.enable_progress_bar && !debug);
 
+    // Initialize Flow Refinement Scheduler
+    std::unique_ptr<IRefiner> flows(nullptr);
```

Comment:
is there any reason why we may not want to do this with all of the other refiners as well?
it would simplify some interfaces